tests/features/environment.py

```diff
@@ -96,6 +98,10 @@ def before_all(context):
 
     context.fixture_data = fixutils.read_fixture_files()
 
+    context.env_config_home = tempfile.mkdtemp(prefix='pgcli_home_')
+
+    os.environ['XDG_CONFIG_HOME'] = context.env_config_home
```

Comment:
Do you have to restore this variable back to previous value after tests are run?

Reply:
I don't see why. Env vars process-only anyway.